### ISSUE TITLE
Check accessibility issues

### ISSUE THREAD

**Issue by marcol13** [marcol13/mkrueger.pl#3]
It is necessary to fulfill requirements related with accessibility (a11y), such as WCAG.

Useful resources:
- https://www.a11yproject.com/checklist/
- WAVE plugin

Potential problems:
- text color on emphasize fragments
- no image labels
- lack of reduced-motion styling
- no `tabindex` on certain elements
